server/cloudapps/autocomplete.go

```diff
@@ -0,0 +1,38 @@
+package cloudapps
+
+type Autocomplete struct {
+	Form
+}
+
+type autocompleteElementProps struct {
+	FlagName string `json:"flag_name"`
```

Comment:
Can we have this be `FieldName` or something similar? Mainly because I want to include positional arguments here

Reply:
`elementProps` already has a `Name` - which is the JSON name - is it not sufficient? Or you want display names for positional arguments (not sure if it makes sense - where would they be displayed?)

Reply:
> elementProps already has a Name - which is the JSON name - is it not sufficient?

The `Name` field works :+1: